AirCasting/SessionViews/SessionCellView.swift

```diff
@@ -71,11 +97,22 @@ private extension SessionCellView {
         }
         .buttonStyle(GrayButtonStyle())
     }
+    
+    func thresholdFor(selectedStream: MeasurementStreamEntity?) -> SensorThreshold? {
+        thresholds.first { threshold in
+            let streamName = selectedStream?.sensorName?
+                .drop { $0 != "-" }
+                .replacingOccurrences(of: "-", with: "")
+                .lowercased()
```

Comment:
drop and replace is the same as in `MeasurementsView.swift`. Maybe it should be moved to an extension?

Reply:
Can't move it to the extension, because in first case I'm doing it on `SensorThreshold` and in the second on `MeasurementStreamEntity`. So I would need to do it twice anyways. 
Do you think it would be a good idea to make a global func for that?

Reply:
When something like this happens it tells us that we should probably create a new object. Currently we're using entities directly which prevents us from doing so, but in the future we should move all `let/var sensorName: String` declarations from being `String` to a custom type like `SensorName` which would implement this logic. For now we can introduce something super simple like:
``` Swift
class SensorNameFormatter {
     func format(_ name:) -> String {
          ...
     }
}
```

and just use it directly here:
``` Swift
let streamName = SensorNameFormatter().format(sensorName)
```

It's not ideal, but will do 🤷‍♂️

Reply:
O, great tip, thanks. I wouldn't come up with such solution :)

Reply:
I'm adding it as a fix for the future then, after the refactor.